package.json

```diff
@@ -72,7 +78,8 @@
     "react": "^15.4.2",
     "react-addons-css-transition-group": "^15.4.2",
     "react-alert": "^1.0.14",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "styled-components": "1.4.3"
```

Comment:
To maintain consistency, should we keep carets in version. `"styled-components": "^1.4.3"`. And if we need to freeze for a reason in future, we freeze them all.